app/templates/Articles/index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @var \App\View\AppView $this
+ * @var iterable<\Cake\Datasource\EntityInterface> $articles
+ */
+?>
+<h1>Articles</h1>
+<p>
+    <?= $this->Html->link("Add Article", ['action' => 'add']) ?>
+</p>
+<table>
+    <tr>
+        <th>Title</th>
+        <th>Created</th>
+        <th>Action</th>
+    </tr>
+
+    <!-- Here's where we iterate through our $articles query object, printing out article info -->
+
+    <?php foreach ($articles as $article): ?>
+        <tr>
+            <td>
+                <?= $this->Html->link($article->title, ['action' => 'view', $article->id]) ?>
+            </td>
+            <td>
+                <?= $article->created->format(DATE_RFC850) ?>
+            </td>
+            <td>
+                <?= $this->Html->link('Edit', ['action' => 'edit', $article->id]) ?>
+                <?= $this->Form->postLink(
+                    'Delete',
+                    ['action' => 'delete', $article->id],
+                    ['confirm' => 'Are you sure?']
+                )
+                    ?>
```

Comment:
```suggestion
                    ['confirm' => 'Are you sure?'])
                ?>
```

The indentation looks weird. Please take care the indentation format as well.

Reply:
I use an indentation extension to manage this kind of issues. For frontend I use `prettier` and a pre-commit hook to avoid these kind of comments. I suggest we can do something similar for the project.

Reply:
Usually, we didn't configure prettier / pre-commit hook to fix the indentation. We used the default setting in IntelliJ. 

I think we experience before the indentation become different when the engineer use different IDE than IntelliJ.